### DISCUSSION TITLE
Help with 2 issues after building KeePassXC in Debian 11

### DISCUSSION THREAD

**Discussion by xcpn** [keepassxreboot/keepassxc#10071] [Q&A] [answered]
I have managed to build KeePassXC in Debian with no networking and only the YubiKey extension enabled.

Followed the [documentation](https://github.com/keepassxreboot/keepassxc/blob/develop/INSTALL.md) to build KeePassXC and although if I run `sudo make install` I end up with a completely funcional build, I still get the following errors when launched via terminal.

- YubiKey: Failed to establish PCSC context.
- YubiKey: PCSC interface is disabled or not initialized.

The YubiKey is recognized by KeePassXC and I can unlock my database, but why are these errors present (libykpers-1-1, libyubikey-dev, libyubikey0 are installed)?

The second issue is when installing KeePassXC as a .deb package. I managed to create a completely functional .deb package, but when installed with `sudo dpkg -i keepassxc.deb` I get dependency problems because they're not already installed. Running `sudo apt-get -f install` afterwards solves the problem, but how can I prevent this issue and have DPKG handle it itself in the first place?

**Comment by droidmonkey** [comment, marked as answer]
The _warnings_ you see are related to NFC support. Ignore them.

We do not provide support for dpkg, you'll have to consult their documentation on how to make a proper package with the required dependencies.

**Reply by xcpn** [reply]
Hey @droidmonkey, quick question.

What folder is used when building KeePassXC, to add the icon in the KeePassXC start menu (where you can create/open/import a database)?

I want to change the icon in the start menu (not the .desktop).

**Reply by droidmonkey** [reply]
On Linux? That uses the .desktop file.

**Reply by xcpn** [reply]
@droidmonkey no, sorry I was not clear.

In Linux, what file in what directory should I replace when building KeePassXC to change the icon in the Welcome Screen?

![11.png](https://github.com/keepassxreboot/keepassxc/assets/134593944/cc541492-9157-4463-99ba-449ec5b73866)

**Reply by droidmonkey** [reply]
https://github.com/keepassxreboot/keepassxc/tree/develop/share/icons/application/scalable/apps

**Reply by xcpn** [reply]
> https://github.com/keepassxreboot/keepassxc/tree/develop/share/icons/application/scalable/apps

So for this specific icon it should be the `keepassxc.svg` file?

**Reply by droidmonkey** [reply]
Possibly, I'm not sure exactly which one is used but that is most likely

**Reply by xcpn** [reply]
Ok thank you, appreciated!
